domains/connie.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "gvalue-04",
+           "email": "",
+           "discord": "1229820339840618506",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.i8mk_73yrOluApbS_s2gO9pL_gIoLP-xv6lRw0KVlvP0QFudw8aIMx_xVKJ8RSXfvMZueFD9VQ3Ma_pASHMNKGY0tIEsqVJo_d_Lzi-FD5kslV-AscUDbl7LPDYza5SriCsUWaXkMXgx_LTpjxH_mJa8UT9Tg282EGRW7OdJySGWslUIbftYw0bZrIGH-kIKyG1nCpkgsXZKOsoz2r5jBO2la1_smRGzStXrbiGGAp8anJMI5q9BejlUFoqmlqXpIn6NPacUJSwXQ2pja8SQgwvWWx2KIMcgDUIrrnb1g1c2Hd3S9I4pgLsQmbXOMmn4wu8mVlKCBqx159PomdX6HA.ppTpXGvEiJ1slqJbtbzGww.Jg6LN8wrDnF-nRPDT_rUFSimdAmSWusgoOoMs45xOy-05VD74i6e9mcXzIS7b8YJzagA9WJuPYWVO5wPxKbdITsdZii-5Vqu2jeplRqQlh4.ZK4ZjA8iuHTreCAkWxQNLg"
+        },
+    
+        "record": {
+            "A": ["198.91.81.13"]
```

Comment:
Please give a preview of your website.

Reply:
it will be a GNU social site, my provider doesnt allow me to see the site without a domain